### ISSUE TITLE
Make Bisq sites reachable via onion address

### ISSUE THREAD

**Issue by cd2357** [bisq-network/bisq#4400]
Make sure all Bisq sites that currently rely on clearnet domains are also reachable via an onion address.

In addition, the sites could advertise the onion address via the `Onion-Location` HTTP header, which would make Tor Browser users be automatically notified of the onion address, each time they visit a Bisq site.

See https://www.torproject.org/releases/tor-browser-95/ for details and screenshots.

See https://community.torproject.org/onion-services/advanced/onion-location/ for Apache and nginx config examples.

**Comment by wiz** [comment]
This issue should be transferred to the ops repo.

What domains don't support Tor? mempool.space certainly does, and this is hard-coded in the pull-down combo box as well.

**Comment by cd2357** [comment]
Yes, mempool.space is reachable via and advertises its onion address in the HTTP headers (so when accessing mempool.space in Tor Browser, the address bar shows a hint ".onion available").

However all the BIsq doamins listed here (https://github.com/bisq-network/proposals/issues/211#issuecomment-669979112) do not.

Technically, they area reachable via Tor -- but having their own onion address would mean traffic never leaves the Tor network.

This would bring more privacy to users + would make Bisq resilient against DNS vulnerabilities (technical or operational, see https://github.com/bisq-network/proposals/issues/211)

> This issue should be transferred to the ops repo.

What's the recommended way to do that? Delete it here and re-add there? Or create new one there, link to this one?
